http/rest-client-reactive/src/test/java/io/quarkus/ts/http/restclient/reactive/OpenShiftTLSRegistryIT.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.ts.http.restclient.reactive;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Tag;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+
+@Tag("QUARKUS-5664")
+@QuarkusScenario
```

Comment:
btw this needs to be `@OpenShiftScenario`, but I think whoever will try to enable it will figure...